ztp/source-crs/PtpConfigDualCardsWpc.yaml

```diff
@@ -0,0 +1,227 @@
+        settings:
+          LocalMaxHoldoverOffSet: 1500
+          LocalHoldoverTimeout: 14400
+          MaxInSpecOffset: 100
+        pins: $e810_pins
+        #  "$iface_master":
+        #    "U.FL2": "0 2"
+        #    "U.FL1": "0 1"
+        #    "SMA2": "0 2"
+        #    "SMA1": "2 1"
+        #  "$iface_master_1":
+        #    "U.FL2": "0 2"
+        #    "U.FL1": "0 1"
+        #    "SMA2": "0 2"
+        #    "SMA1": "1 1"
+        ublxCmds:
+          - args: #ubxtool -P 29.20 -z CFG-HW-ANT_CFG_VOLTCTRL,1
+              - "-P"
+              - "29.20"
+              - "-z"
+              - "CFG-HW-ANT_CFG_VOLTCTRL,1"
+            reportOutput: false
+          - args: #ubxtool -P 29.20 -e GPS
+              - "-P"
+              - "29.20"
+              - "-e"
+              - "GPS"
+            reportOutput: false
+          - args: #ubxtool -P 29.20 -d Galileo
+              - "-P"
+              - "29.20"
+              - "-d"
+              - "Galileo"
+            reportOutput: false
+          - args: #ubxtool -P 29.20 -d GLONASS
+              - "-P"
+              - "29.20"
+              - "-d"
+              - "GLONASS"
+            reportOutput: false
+          - args: #ubxtool -P 29.20 -d BeiDou
+              - "-P"
+              - "29.20"
+              - "-d"
+              - "BeiDou"
+            reportOutput: false
+          - args: #ubxtool -P 29.20 -d SBAS
+              - "-P"
+              - "29.20"
+              - "-d"
+              - "SBAS"
+            reportOutput: false
+          - args: #ubxtool -P 29.20 -t -w 5 -v 1 -e SURVEYIN,600,50000
+              - "-P"
+              - "29.20"
+              - "-t"
+              - "-w"
+              - "5"
+              - "-v"
+              - "1"
+              - "-e"
+              - "SURVEYIN,600,50000"
+            reportOutput: true  
+          - args: #ubxtool -P 29.20 -p MON-HW
+              - "-P"
+              - "29.20"
+              - "-p"
+              - "MON-HW"
+            reportOutput: true
+    ts2phcOpts: " "
+    ts2phcConf: |
+      [nmea]
+      ts2phc.master 1
+      [global]
+      use_syslog  0
+      verbose 1
+      logging_level 7
+      ts2phc.pulsewidth 100000000
+      #GNSS module s /dev/ttyGNSS* -al use _0
+      #cat /dev/ttyGNSS_1700_0 to find available serial port
+      #example value of gnss_serialport is /dev/ttyGNSS_1700_0
+      ts2phc.nmea_serialport $gnss_serialport
+      leapfile  /usr/share/zoneinfo/leap-seconds.list
+      [$iface_master]
+      ts2phc.extts_polarity rising
+      ts2phc.extts_correction 0
+      [$iface_master_1]
+      ts2phc.extts_polarity rising
+      ts2phc.extts_correction 0
+      #this is a measured value in nanoseconds to compensate for SMA cable delay
+      ts2phc.extts_correction -25
+    ptp4lConf: |
+      [$iface_master]
+      masterOnly 1
+      [$iface_master_1]
+      masterOnly 1
+      [$iface_master_2]
+      masterOnly 1
+      [$iface_master_3]
```

Comment:
There should be $iface_master_1_1 and $iface_master_1_2
$iface_master_1 of $iface_master is confused with 2nd interface $iface_master_1